2gp-demo/sfdx-project.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
this file contains specific namespace and org IDs. is it necessary?

Reply:
The specific namespace and org IDs are there only while I'm working on this.

When this repo goes live they will be removed and what's in the `sfdx-project-baseline.json` file will be used instead.